src/gsi/gsi_dbzOper.F90

```diff
@@ -99,8 +103,30 @@ subroutine setup_(self, lunin, mype, is, nobs, init_pass,last_pass)
     character(len=len_isis   ):: isis
     integer(i_kind):: nreal,nchanl,ier,nele
     logical:: diagsave
+    integer(i_kind):: lu_diag
+    character(128):: diag_file
+    character(80):: string
+
+    if(nobs == 0) then
+
+       if(mype == 0) then
```

Comment:
Lines 112-115 can be deleted, if necessary.  They were for debugging purposes only.